clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/handlers/BasicGoogleDeployHandler.groovy

```diff
@@ -375,8 +375,7 @@ class BasicGoogleDeployHandler implements DeployHandler<BasicGoogleDeployDescrip
     }
 
     List<InstanceGroupManagerAutoHealingPolicy> autoHealingPolicy =
-      description.autoHealingPolicy?.healthCheck
-      ? [new InstanceGroupManagerAutoHealingPolicy(
+      description.autoHealingPolicy?.healthCheck ? [new InstanceGroupManagerAutoHealingPolicy(
```

Comment:
Is this reformatting intentional? Don't see a code change for this line.

Reply:
Intellij was complaining about the `?` on the second line

Reply:
Yeah. The folks here make fun of me for formatting it this way. I prefer the original style, but can live with it either way. I do typically try to avoid reformatting lines that don't have code changes unless it really affects readability.

Reply:
I can adjust it. I might have some intellij setting that was flagging it as an error and so wouldn't compile.